lab_7_llm/assets/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Checkout the files from 7 lab to the master

Reply:
done